spec/bashly/commands/generate_spec.rb

```diff
@@ -51,6 +51,34 @@
     end
   end
 
+  context "with --quiet" do
+    let(:cli_script) { "#{target_dir}/cli" }
+
+    before do
+      reset_tmp_dir
+      success = system "mkdir -p #{source_dir} && cp lib/bashly/templates/bashly.yml #{source_dir}/bashly.yml"
+      expect(success).to be true
+    end
+
+    it "generates the cli script" do
+      expect { subject.run %w[generate --quiet] }.to output_approval('cli/generate/quiet')
+      expect(File).to exist(cli_script)
+    end
+
+    context "when source files already exist" do
+      before do
+        expect { subject.run %w[generate --quiet] } #.to output_nothing
```

Comment:
The rspec_approvals gem [doesn't seem to like empty output](https://github.com/DannyBen/rspec_approvals/blob/v0.9.2/lib/rspec_approvals/matchers/base.rb#L16), which is awkward in this situation. Since you're the creator of both gems I figured you'd have an idea about what to do here.

Reply:
Well, it is by design. An empty output can be tested with `.to be_empty` and there is no need for an approval matcher. So, it is usually a mistake when you use any of the approval matches, with an empty output. But don't worry about it, I will take a deeper look in the spec and fix accordingly.